### ISSUE TITLE
modified device analyzer can't scan explosive implant

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#11053]
(WEB REPORT BY: realestestate REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)

20e8385cf18921c12e1b48770f334422bc9ddc20

> General description of the issue
> the job specific Syndicate device analyzer doesn't scan explosive implant, essentially acting as a throw-away for 4 TC if you used the other 6 to get an Explosive implant. if anything, the item should warn you on what it can and can't scan, rather than being literal waste of fucking telecrystals.
> 
> What you expected to happen
> be able to scan and mass produce explosive implants. it's the fucking point of the damn item
> 
> What actually happened
> "The device analyzer can't seem to scan the implanter (E)!"
> 
> Steps to reproduce if possible
> be mechanic syndicate agent. use uplink to buy modified device analyzer and explosive implant. try to scan explosive implant using modified device analyzer. throw modified device analyzer away because it's literally worthless now.

**Comment by charmstruck** [comment]
> throw modified device analyzer away because it's literally worthless now.

Can't you scan guns with it, unlike the normal DA?

**Comment by kernikov** [comment]
Yeah, and you can scan most tator gear like thermals and e-swords.

**Comment by ComicIronic** [comment]
I give this a honk/10.
And to fix this, you only need to set the explosive implant to have some tech levels and some mats, both of which are trivial var changes.

**Comment by Intigracy** [comment]
The implanter is like a syringe though, tou'd need to change the analyzer to scan the implants inside of the implanter instead of the implanter itself.
